rmw_opensplice_cpp/src/rmw_node.cpp

```diff
@@ -46,7 +46,7 @@ rmw_node_t *
 rmw_create_node(
   rmw_context_t * context,
   const char * name, const char * namespace_, size_t domain_id,
-  const rmw_node_security_options_t * security_options)
+  const rmw_node_security_options_t * security_options, bool localhost_only)
```

Comment:
The implementation block should contain `(void)localhost_only` to avoid a compiler warning.